astropy/modeling/functional_models.py

```diff
@@ -545,26 +547,29 @@ class Sine1D(Fittable1DModel):
     -----
     Model formula:
 
-        .. math:: f(x) = A \\sin(2 \\pi f x)
+        .. math:: f(x) = A \\sin(2 \\pi f x + 2 \\pi p)
     """
 
     amplitude = Parameter(default=1)
     frequency = Parameter(default=1)
+    phase = Parameter(default=0)
 
     @staticmethod
-    def evaluate(x, amplitude, frequency):
+    def evaluate(x, amplitude, frequency, phase):
         """One dimensional Sine model function"""
 
-        return amplitude * np.sin(2 * np.pi * frequency * x)
+        return amplitude * np.sin(2 * np.pi * frequency * x + np.pi * phase)
```

Comment:
Missing factor of 2?
Incidentally, it's common to define a constant `_TWOPI = 2 * np.pi` (or you could call it `_TAU`).  Maybe that should be used here (and in any other relevant code in this module).